ray-operator/controllers/ray/common/pod_test.go

```diff
@@ -1300,6 +1311,19 @@ func TestGenerateRayStartCommand(t *testing.T) {
 			},
```

Comment:
can you update the test case "HeadNode with multiple custom accelerators" to use the correct container resource name for TPUs?

Reply:
Sure, everything after the first custom accelerator gets ignored so it doesn't look at it but I can change it for consistency.

Reply:
Done in 8a53e4d, this commit also removes `mockCustomAcceleratorToRayResourceMap` since it's no longer needed/used as the `google.com/tpu` mapping is added in `customAcceleratorToRayResourceMap`.